src/plasmapy/particles/particle_class.py

```diff
@@ -783,6 +785,19 @@ def _assign_atom_attributes(self) -> None:
 
         categories.add(this_element["category"])
 
+        try:
```

Comment:
One more addition to this logic: I don't think the isotope number changes the ionization energy, but right now I think this code fails to find the ionization energy if you specify the isotope, e.g. 

```python
> Particle('C +5').ionization_energy.to(u.eV)
Out[13]: 489.99 eV

 

>Particle('C-12 +5').ionization_energy.to(u.eV)
Traceback (most recent call last):

  Cell In[12], line 1
    Particle('C-12 +5').ionization_energy.to(u.eV)

  File ~\GitHub\PlasmaPy\src\plasmapy\particles\particle_class.py:1900 in ionization_energy
    raise MissingParticleDataError(

MissingParticleDataError: The ionization energy of C-12 5+ is not available.
``` 

I think the correct behavior should be that, if the particle is an isotope, the lookup symbol should be the element/ion without the isotope specified? e.g. the following should work

```python
>Particle('U-235 +5').ionization_energy == Particle('U-238 +5').ionization_energy
True
```

Reply:
Read one of your other comments before this one, and I think this one answers my question 😂 

Thanks for checking me here - this was due to me having an incomplete understanding of particle physics and playing it on the safe side

Reply:
No problem, I had to look this up too!

Reply:
This should be fixed now in the latest version